apk-generator/v2/app/static/js/main.js

```diff
@@ -245,4 +255,8 @@ $form.submit(function (e) {
         .catch(function () {
             showError();
         });
+    
+    socket.on('message', function(message) {
```

Comment:
you mean defining it within where socket is connected?
also apart from this it was working fine? 
now i should just present a success or fail in front of it right?